### PR TITLE
LOK-2404: Encode interface specifics in SNMP Result Metrics as labels

### PR DESCRIPTION
**Pull request by aurang-zeb313**
## Description
Search code for LOK-2404 and remove all marked named values from proto including all use-cases. 
Use labels as a replacement.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2404

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: minion/plugins/snmp-plugin/src/main/java/org/opennms/horizon/minion/snmp/SnmpCollectionSet.java [hunk 1]
```diff
@@ -86,8 +86,8 @@ private static SnmpResultMetric mapResult(
                 .setInstance(result.getInstance().toString())
                 .setValue(mapValue(result.getValue()))
                 .setAlias(alias)
-                .setIfName(ifName)
-                .setIpAddress(ipAddress)
+                // .setIfName(ifName)
+                // .setIpAddress(ipAddress)
```
Comment (smunir-onms) [inline minion/plugins/snmp-plugin/src/main/java/org/opennms/horizon/minion/snmp/SnmpCollectionSet.java:90]: please remove commented code

**Review by fooker — CHANGES_REQUESTED** [review]
This seems to be quite similar to #1812.

**Comment by smunir-onms** [comment]
> LGTM, needs to validate, I'll do it on my machine

validation recording attached;
[Screencast from 01-05-2024.webm](https://github.com/OpenNMS-Cloud/lokahi/assets/157733753/81df4b01-2425-4a7a-a709-b7700c28a330)

**Review by smunir-onms — COMMENTED** [review]
LGTM, needs to validate the changes.

**Review by fooker — APPROVED** [review]

**Pull request merged**